plugins/scaffolder-backend/sample-templates/remote-templates.yaml

```diff
@@ -7,3 +7,9 @@ spec:
   type: url
   targets:
     - https://github.com/spotify/cookiecutter-golang/blob/master/template.yaml
+    - https://github.com/backstage/software-templates/blob/main/scaffolder-templates/create-react-app/template.yaml
```

Comment:
I'm unsure of this change though? The only time that these templates would be referenced is when we're working from the monorepo, so why not just make it local anway? Just feels like an uneeded step?

Reply:
Reading the conversation in https://github.com/backstage/backstage/issues/3383

I think there is some value in giving our sample templates a bit more "visibility". The [software template marketplace](https://github.com/backstage/software-templates) can be a good place where new users explore the templates to seek inspiration for writing their templates and community to share back templates, ultimately adding value for the rest. Hence we thought, let's move out our sample templates from `plugins/scaffolder-backend` in the mono-repo to a more apparent location.

Reply:
Ah sorry - I totally missed that we were moving the templates that we read from the main repo there too.